source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -234,10 +234,11 @@ void EnvoyQuicClientStream::OnInitialHeadersComplete(bool fin, size_t frame_len,
   }
   saw_regular_headers_ = false;
   quic::QuicRstStreamErrorCode transform_rst = quic::QUIC_STREAM_NO_ERROR;
+  auto client_session = static_cast<EnvoyQuicClientSession*>(session());
```

Comment:
@danzh2010  Can the session() call returns a nullptr by any chance?

Reply:
No.

Reply:
Thanks for confirming!